modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/ManagementToolbarTag.java

```diff
@@ -1255,9 +1254,11 @@ protected int processStartTag() throws Exception {
 					new Object[] {getItemsTotal()}));
 
 			jspWriter.write("<strong>");
-			jspWriter.write(
-				HtmlUtil.escape(
-					StringBundler.concat(" ", '"', searchValue, '"')));
+			jspWriter.write(HtmlUtil.escape(StringPool.SPACE));
```

Comment:
@sergiojimcos overall LGTM. I just had a doubt about using `HtmlUtil.escape`, but after thinking about it since quotes and spaces are usually "escaped" in HTML I guess it doesn't hurt. 
I'll let the others share their thoughts on this.

Reply:
On a second thought @sergiojimcos, there's no need to "escape" those things, see [this](https://github.com/liferay/liferay-portal/blob/be4e29241f15319c98af8d2ac975e28ce3b54b25/modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/PaginationBarTag.java#L217) (and the entire `processStartTag` method). Sorry for the confusion.